repos/fdbt-site/src/pages/api/defineCaps.ts

```diff
@@ -0,0 +1,93 @@
+import { NextApiResponse } from 'next';
+import { FULL_CAPS_ATTRIBUTE, CAPS_DEFINITION_ATTRIBUTE } from '../../constants/attributes';
+import { getCapByNocAndId } from '../../data/auroradb';
+import { getAndValidateNoc, redirectTo, redirectToError } from '../../utils/apiUtils/index';
+import { CapInfo, CapSelection, CapsDefinitionWithErrors, NextApiRequestWithSession, PremadeCap } from 'src/interfaces';
+import { updateSessionAttribute } from '../../../src/utils/sessions';
+
+const getCapContent = async (
+    cap: number | undefined,
+    noc: string,
+): Promise<
+    | {
+          dbCap: PremadeCap;
+          caps: CapInfo;
```

Comment:
Formatting here looks very strange - is this how its meant to look?

Reply:
yes same as time restrictions